spec/lib/reporting/fraud_metrics_lg99_report_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
+require 'reporting/fraud_metrics_lg99_report'
+
+RSpec.describe Reporting::FraudMetricsLg99Report do
+  let(:time_range) { Date.new(2022, 1, 1).all_month }
+  let(:expected_lg99_metrics_table) do
+    [
+      ['Metric', 'Total'],
+      ['Unique users seeing LG-99', '5'],
+    ]
+  end
+
+  subject(:report) { Reporting::FraudMetricsLg99Report.new(time_range:) }
+
+  before do
+    cloudwatch_client = double(
+      'Reporting::CloudwatchClient',
+      fetch: [
+        { 'user_id' => 'user1', 'name' => 'IdV: Verify please call visited' },
+        { 'user_id' => 'user1', 'name' => 'IdV: Verify setup errors visited' },
+
+        { 'user_id' => 'user1', 'name' => 'IdV: Verify please call visited' },
```

Comment:
this is `user1` with an event it's already seen, was this to check that uniq'ing worked? If it's meant to be a dupe I think we should move it up closer to the line it duplicates

If not, let's remove?